sketch_map_tool/routes.py

```diff
@@ -105,13 +105,18 @@ def digitize_results_post() -> Response:
     if "file" not in request.files:
         return redirect(url_for("digitize"))
     files = request.files.getlist("file")
+    max_nr_simultaneous_uploads = 100
+    if len(files) > max_nr_simultaneous_uploads:
+        raise UploadLimitsExceededError(
+            f"You can only upload up to {max_nr_simultaneous_uploads} files at once."
+        )
```

Comment:
Since we do not have much experience what a good maximum number of uploaded files is, it would be great to be able to change the value on the fly without changing the code. As you suggested, having this variable in the config would be great.